drivers/w1/w1_net.c

```diff
@@ -346,11 +346,15 @@ int w1_skip_rom(const struct device *dev, const struct w1_slave_config *config)
 
 static int reset_select(const struct device *dev, const struct w1_slave_config *config)
 {
+#if W1_IGNORE_SLAVE_COUNT
+	return match_rom(dev, config);
+#else
 	if (w1_get_slave_count(dev) > 1) {
```

Comment:
First, you need to prefix with `CONFIG_`.
Also, you could do instead:

```
if (!IS_ENABLED(CONFIG_W1_IGNORE_SLAVE_COUNT) && w1_get_slave_count(dev) > 1) {
```

Reply:
True, done.